weed/command/scaffold/filer.toml

```diff
@@ -60,6 +60,8 @@ connection_max_idle = 2
 connection_max_open = 100
 connection_max_lifetime_seconds = 0
 interpolateParams = false
+#     [username[:password]@][protocol[(address)]]/dbname[?param1=value1&...&paramN=valueN]
+dsn = "root@tcp(localhost:3306)/seaweedfs?collation=utf8mb4_bin"
```

Comment:
move the new section to after line 53, and add comments that this dsn will take priority over "hostname, port, username, password, database".